neuron/src/lib/Neuron/Zettelkasten/Zettel.hs

```diff
@@ -42,7 +42,7 @@ import Text.Show (Show (show))
 -- It does not care about the relationship *between* those zettels; for that use `GraphQuery`.
 data ZettelQuery r where
   ZettelQuery_ZettelByID :: ZettelID -> Connection -> ZettelQuery Zettel
-  ZettelQuery_ZettelsByTag :: [TagPattern] -> Connection -> ZettelsView -> ZettelQuery [Zettel]
+  ZettelQuery_ZettelsByTag :: [TagPattern] -> Maybe Int -> Connection -> ZettelsView -> ZettelQuery [Zettel]
```

Comment:
And I *assume* this is supposed to affect the view only; and not the graph. Because, the backlinks of result zettels would still show the zettel containing the query in its backlink.

Reply:
> Because, the backlinks of result zettels would still show the zettel containing the query in its backlink

That is a good point. The current implementation does what I would expect it to: It creates links to the show zettels only. I.e. if I write `[[[z:zettels?tag=daily/**&limit=2&timeline]]]` it shows the latest two zettels in my timeline and creates folgezettel relationships to them, but to no other zettel. So it affects the graph and that I how I would expect it and what is most intuitive. *View only* queries should maybe use a different format to make it clear that they don't affect the graph?

> the Maybe Int should go inside the ZettelsView type
> While we are at it, see if you can use Natural instead of Int.

That makes sense. I'll get to that this evening (european time).

Reply:
> the Maybe Int should go inside the ZettelsView type
> While we are at it, see if you can use Natural instead of Int.

These two points are done.